pkg/ccl/changefeedccl/changefeedbase/settings.go

```diff
@@ -252,3 +252,13 @@ var EventConsumerWorkerQueueSize = settings.RegisterIntSetting(
 	int64(util.ConstantWithMetamorphicTestRange("changefeed.event_consumer_worker_queue_size", 16, 0, 16)),
 	settings.NonNegativeInt,
 ).WithPublic()
+
+// EventConsumerDispatcherBufferSize specifies the maximum number of events the dispatcher can buffer.
+var EventConsumerDispatcherBufferSize = settings.RegisterIntSetting(
+	settings.TenantWritable,
+	"changefeed.event_consumer_dispatcher_buffer_size",
+	"if changefeed.event_consumer_workers is enabled, this setting sets the maxmimum number of events"+
+		"which can be buffered by the dispatcher",
+	int64(util.ConstantWithMetamorphicTestRange("changefeed.event_consumer_dispatcher_buffer_size", 64, 0, 64)),
+	settings.NonNegativeInt,
+).WithPublic()
```

Comment:
this strikes me as a very, very low level setting; almost certainly not "WithPublic" -- it might even just be a metamorphic constant...

Reply:
Removed `WithPublic`, it just came along since I copied the similar nprocs setting.